server/charts/kafka/templates/zookeeper-stateful-set.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
 spec:
   serviceName: zookeeper
   replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "kafka.fullname" . }}
```

Comment:
zookeeper.fullname

Reply:
Should this value be the same as template/metadata/labels/app's value, that is,
{{ template: "kafka.fullname" .}} ?

Reply:
Ah you are correct. I forgot that we use the same name for Kafka and zookeeper. Makes sense. I will merge it now. Thanks for the PR.